### PR TITLE
Removal of mentions of Nodejs12

### PR DESCRIPTION
**Pull request by Cortey**
**Description**
Removing all mentions of nodejs12 from documentation.

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/15771

### CODE REVIEW

File: docs/01-overview/main-areas/serverless/svls-02-from-code-to-function.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Pick the programming language for the Function and decide where you want to keep
 
 ## Runtimes
 
-Functions support multiple languages by using the underlying execution environments known as runtimes. Currently, you can create both Node.js (12 & 14) and Python (3.9) Functions in Kyma.
+Functions support multiple languages by using the underlying execution environments known as runtimes. Currently, you can create both Node.js (14) and Python (3.9) Functions in Kyma.
```
Comment (anoipm) [inline docs/01-overview/main-areas/serverless/svls-02-from-code-to-function.md:9]: What with Node.js 16?

File: docs/01-overview/main-areas/serverless/svls-04-development-toolkit.md [hunk 2]
```diff
@@ -10,6 +10,6 @@ To start developing your first Functions, you need:
 - [**kubectl**](https://kubernetes.io/docs/reference/kubectl/kubectl/), the Kubernetes command-line tool, for running commands against clusters
 - Development environment of your choice:
    - **Kyma CLI** to easily initiate inline Functions or Git Functions locally, run, test, and later apply them on the clusters
-   - **Node.js** (v12 or v14) or **Python** (v3.9)
+   - **Node.js** (v14) or **Python** (v3.9)
```
Comment (anoipm) [inline docs/01-overview/main-areas/serverless/svls-04-development-toolkit.md:13]: What with Node.js 16?

File: docs/05-technical-reference/svls-04-git-source-type.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 title: Git source type
 ---
 
-Depending on a runtime you use to build your Function (Node.js 12, Node.js 14, or Python 3.9), your Git repository must contain at least a directory with these files:
+Depending on a runtime you use to build your Function (Node.js 14, or Python 3.9), your Git repository must contain at least a directory with these files:
```
Comment (anoipm) [inline docs/05-technical-reference/svls-04-git-source-type.md:5]: What with Node.js 16?

File: docs/05-technical-reference/svls-08-function-specification.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 title: Function's specification
 ---
 
-Serverless in Kyma allows you to create Functions in both Node.js (v12 & v14) and Python (v3.9). Although the Function's interface is unified, its specification differs depending on the runtime used to run the Function.
+Serverless in Kyma allows you to create Functions in both Node.js (v14) and Python (v3.9). Although the Function's interface is unified, its specification differs depending on the runtime used to run the Function.
```
Comment (anoipm) [inline docs/05-technical-reference/svls-08-function-specification.md:5]: What with Node.js 16?

**Review by grego952 — APPROVED** [review]

**Pull request merged**